### ISSUE TITLE
Consider including Lodash in the project

### ISSUE THREAD

**Issue by justin808** [shakacode/react-webpack-rails-tutorial#124]
Take a look at this:
https://github.com/shakacode/react-webpack-rails-tutorial/pull/123/files#r42269891

The take away is this:

```javascript
Array.from(document.getElementsByTagName('meta')).find(tag => tag.name === 'csrf-token')
```

vs with Lodash:

```javascript
_.find(document.querySelectorAll('meta'), 'name', 'csrf-token')
```

I find lodash to be my swiss army knife when coding JS, and it fits in well with React.

In fact, Lodash is already an implicit dependency.

https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/client/npm-shrinkwrap.json#L2669

Anybody second me?

@alexfedoseev @samnang @josiasds @mapreal19 @dylangrafmyre @rstudner?

**Comment by samnang** [comment]
:+1:

**Comment by justin808** [comment]
https://www.youtube.com/watch?v=Pq1RD22W7B4#t=55s

I vote for lodash.

CC: @jdalton

**Comment by justin808** [comment]
More on lodash: https://github.com/Khan/style-guides/pull/28/files#files

My personal preference is to lean on lodash for consistency. It's a lot of work to keep track of both, and like using the linters, I prefer consistency.

**Comment by justin808** [comment]
@josiasds I put your name on this one. Should be quick. 

@alexfedoseev if we put lodash in the vendor bundle, I'm pretty sure that parts we don't user don't get stripped out. That's a need for jQuery. I'm thinking we could have a generated/vendor-optimized.js bundle that has all the other libraries we include with unused stripped out. 

I'd like to be able to see the project, non-vendor, code in it's own generated file, as that's useful for explaining what Webpack does.

**Comment by robwise** [comment]
I know it's nice to have the webpage download as small as possible, but how many KB are we talking about here? If we use half of the lodash library and leave the other half unused, we'd be foregoing, what, 9KB of savings by just importing all of it?

**Comment by justin808** [comment]
Fixed in #147
